docs/swagger/traPortfolio.v1.yaml

```
@@ -842,23 +842,27 @@ components:
           type: array
           description: コンテスト参加者
           items:
-            $ref: '#/components/schemas/ContestMember'
+            $ref: "#/components/schemas/ContestMember"
     EventLevel:
       type: integer
       title: EventLevel
       description: |-
         公開範囲設定
-        0 全て公開
-        1 イベント企画者の名前を伏せて公開
+        0 イベント企画者の名前を伏せて公開
+        1 全て公開
+        2 外部に非公開
       enum:
         - 0
         - 1
+        - 2
       x-enum-varnames:
-        - onymous
-        - anonymous
+        - EventLevelAnonymous
+        - EventLevelPublic
+        - EventLevelPrivate
```

```suggestion
        - Anonymous
        - Public
        - Private
```
EventLevel.EventLevelAnonymousみたいに冗長になるからこっちで 🙏

---

:haakusimasita: